packages/landing/src/landing.scss

```diff
@@ -145,20 +145,16 @@ header {
     }
   }
 
-  .pt-features {
-    display: flex;
+  .pt-feature-column:nth-child(1) {
+    float: left;
+    width: calc((100% - (#{$pt-grid-size} * 4)) / 2);
+    max-width: $pt-grid-size * 36.4;
   }
 
-  .pt-feature-column {
-    flex: 0 0 50%;
-
-    &:not(:first-child) {
-      padding-left: $pt-grid-size * 4;
-    }
-
-    &:not(:last-child) {
-      padding-right: $pt-grid-size * 4;
-    }
+  .pt-feature-column:nth-child(2) {
+    float: right;
+    width: calc((100% - (#{$pt-grid-size} * 4)) / 2);
+    max-width: $pt-grid-size * 36.4;
```

Comment:
2/3 props are identical between these selectors. pull up to `.pt-feature-column` itself